packages/expo-splash-screen/ios/EXSplashScreen/EXSplashScreenViewNativeProvider.m

```diff
@@ -35,8 +35,32 @@ - (nonnull UIView *)createSplashScreenView
     }
     return views.firstObject;
   }
-  
-  @throw [NSException exceptionWithName:@"ERR_NO_SPLASH_SCREEN" reason:@"Couln't locate neither 'SplashScreen.storyboard' file nor 'SplashScreen.xib' file. Create one of these in the project to make 'expo-splash-screen' work (https://github.com/expo/expo/tree/master/packages/expo-splash-screen#-configure-ios)." userInfo:nil];
+
+  // TODO (@bbarthec): handle fallbacking to default RN Launch Screen view better
+  // @throw [NSException exceptionWithName:@"ERR_NO_SPLASH_SCREEN" reason:@"Couln't locate neither 'SplashScreen.storyboard' file nor 'SplashScreen.xib' file. Create one of these in the project to make 'expo-splash-screen' work (https://github.com/expo/expo/tree/master/packages/expo-splash-screen#-configure-ios)." userInfo:nil];
+
+  return [self createSplashScreenViewFallback];
+}
+
+- (nonnull UIView *)createSplashScreenViewFallback
+{
+  UIView *view;
+  NSArray *views;
+  @try {
+    NSString *launchScreen = (NSString *)[[NSBundle mainBundle] objectForInfoDictionaryKey:@"UILaunchStoryboardName"] ?: @"LaunchScreen";
+    views = [[NSBundle mainBundle] loadNibNamed:launchScreen owner:self options:nil];
+  } @catch (NSException *_) {
+    NSLog(@"LaunchScreen.xib is missing. Unexpected loading behavior may occur.");
```

Comment:
Would we want to use `UMLog…`? If it didn't break, `UMWarn` should show a warning to the user, which may be something we may want to do.